apps/server/src/modules/lesson/service/lesson.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@nestjs/common';
+import { EntityId, Lesson } from '@shared/domain';
+import { LessonRepo } from '@shared/repo';
+import { FileParamBuilder, FilesStorageClientAdapterService } from '@src/modules/files-storage-client';
+
+@Injectable()
+export class LessonService {
+	constructor(
+		private readonly lessonRepo: LessonRepo,
+		private readonly filesStorageClientAdapterService: FilesStorageClientAdapterService
+	) {}
+
+	async deleteLesson(lesson: Lesson, jwt: string): Promise<void> {
```

Comment:
@SevenWaysDP Darf der Lessonservice von dem jwt wissen? Oder Wollen wir hier die fertigen params reingeben?

Reply:
Aktuell sieht es so aus als ob der fileStorageClient in Zukunft kein JWT benötigt und deswegen wird sich das Handlin an dieser Stelle sowieso noch ändern. Bleibt so wie aktuell implementiert.